src/proposalBuilder/executeAction.jsx

```diff
@@ -4,9 +4,18 @@ import { AiOutlineCheck } from 'react-icons/ai';
 
 import { useTX } from '../contexts/TXContext';
 import { InactiveButton } from './proposalActionPrimitives';
+import useCanInteract from '../hooks/useCanInteract';
 
-const ExecuteAction = ({ proposal, executeTX, argsOverride }) => {
+const ExecuteAction = ({
+  proposal,
+  executeTX,
+  argsOverride,
+  disabledOverride,
+}) => {
   const { submitTransaction } = useTX();
+  const { canInteract } = useCanInteract({
```

Comment:
the cards have a canInteract threaded through as props. Would probably be best to use that.